components/automate-ui/src/app/page-components/insight-attributes-dropdown/insight-attributes-dropdown.component.ts

```diff
@@ -11,8 +16,8 @@ export class InsightAttributesDropdownComponent implements OnInit {
 
   @Input() saveAsDefault = true;
   @Input() lastSelectedOptions: string[] = []; // these are filter ids
-  @Output() onUpdateFilters: EventEmitter<any> = new EventEmitter();
-  @Output() onToggleMenu: EventEmitter<any> = new EventEmitter();
+  @Output() onUpdateFilters: EventEmitter<FilterUpdate> = new EventEmitter();
```

Comment:
To complete this improvement, should also use the type on the receiving end:
https://github.com/chef/automate/blob/8b4dd26cf00441b8746e43c272c102c8c55df3d9/components/automate-ui/src/app/modules/desktop/insight/insight.component.ts#L49
==>
```
  public updateFilters(event: FilterUpdate) {
```